apps/google-analytics-4/frontend/src/components/config-screen/HomeAnalytics.styles.ts

```diff
@@ -0,0 +1,27 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { googleAnalyticsBrand } from 'components/common/GoogleAnalyticsIcon';
+
+export const styles = {
```

Comment:
Common Pattern Comment: For complex components, separate logic and styling (Credit: Lillian Bitner)